src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -159,11 +159,11 @@ static ucs_status_t
 uct_rdamcm_cm_ep_create_reserved_qpn(uct_rdmacm_cm_ep_t *cep,
                                      struct uct_rdmacm_cm_qp_context *ctx)
 {
+    ucs_status_t status = UCS_OK;
     struct uct_rdmacm_cm_reserved_qpn_blk *blk;
     uint32_t qpns_per_obj;
-    ucs_status_t status = UCS_OK;
 
-    qpns_per_obj = 1 << ctx->reserved_qpn_ctx.log_reserved_qpn_granularity;
+    qpns_per_obj = uct_rdmacm_cm_get_qpns_per_obj(ctx);
```

Comment:
pls refactor to `uct_rdmacm_cm_get_reserved_qpns_per_obj(ctx->reserved_qpn_ctx)`